src/scripts/sync_subclassof.py

```diff
@@ -452,11 +446,7 @@ def sync_subclassof(
     df3.to_csv(outpath_added, sep='\t', index=False)
 
     # Case 5: SCR is direct in Mondo, and not in the source at all
-    rels_mondo_raw_all: List[RELATIONSHIP] = [x for x in mondo_db.relationships(
-        subjects=list(set(ss_df['subject_id'])), predicates=['rdfs:subClassOf'])]
-    rels_mondo_mondo_all = set(  # filter non-source parents
-        [x for x in rels_mondo_raw_all if any([x[2].startswith(y) for y in MONDO_PREFIX_MAP.keys()])])
-    df5 = pd.DataFrame([{'subject_id': x[0], 'object_id': x[2]} for x in rels_mondo_mondo_all])
+    df5 = pd.DataFrame([{'subject_id': x[0], 'object_id': x[2]} for x in rels_direct_mondo_mondo])
```

Comment:
@joeflack4 what change in the subclass data files does this make?

Reply:
Ah, that is what is mentioned in the OP, and I mentioned in passing at meeting possibly before the holidays:

> One refactor here removes a redundant variable `rels_mondo_mondo_all`  and replaces it with `rels_direct_mondo_mondo`. I tested and confirmed these variables were equivalent.

So there are no changes in outputs.